### PR TITLE
5 get gym member users endpoint

### PR DESCRIPTION
**Pull request by Scott-Borecki**
- Changes Implemented:
  - #53 
    - Included the `gym_member_count` in `json_hash[:meta][:gym_member_count]`

- Quality Control Checklist:
  - [X] Code adheres to Rubocop styling (if unavoidable infractions, please clarify below)
  - [X] 100% SimpleCov test coverage (if below, please clarify below)
  - [X] Last Commit passes Travis build

- Blockers (if applicable):

- Next Steps & Additional Notes:

### CODE REVIEW

**Comment by Scott-Borecki** [comment]
Here's an example output hash from the endpoint: 

```
{:data=>
  [{:id=>"9103",
    :type=>"user",
    :attributes=>
     {:email=>"karma@mueller.com",
      :google_id=>"3dd2wh0igxl077dayldzz",
      :google_image_url=>"https://robohash.org/delectusrationetemporibus.png?size=300x300&set=set1",
      :zip_code=>"19852",
      :summary=>"It is impossible to begin to learn that which one thinks one already knows.",
      :goal=>"Lose Weight",
      :availability_morning=>true,
      :availability_afternoon=>true,
      :availability_evening=>true,
      :full_name=>"Kirk Vandervort"}},
   {:id=>"9104",
    :type=>"user",
    :attributes=>
     {:email=>"joycelyn@klein-lemke.co",
      :google_id=>"mh5n3xkw4x24tkdq22hpi",
      :google_image_url=>"https://robohash.org/sitoptiofugit.png?size=300x300&set=set1",
      :zip_code=>"28037",
      :summary=>"The virtue of justice consists in moderation, as regulated by wisdom.",
      :goal=>"Maintain Weight",
      :availability_morning=>true,
      :availability_afternoon=>true,
      :availability_evening=>false,
      :full_name=>"The Hon. Rhoda Quigley"}},
   {:id=>"9105",
    :type=>"user",
    :attributes=>
     {:email=>"arnoldo@mayer.info",
      :google_id=>"p69op2gvdeityc3mmqlex",
      :google_image_url=>"https://robohash.org/innecessitatibusmolestias.png?size=300x300&set=set1",
      :zip_code=>"05342",
      :summary=>"Love is composed of a single soul inhabiting two bodies.",
      :goal=>"Maintain Weight",
      :availability_morning=>false,
      :availability_afternoon=>false,
      :availability_evening=>false,
      :full_name=>"Heike Beahan"}},
   {:id=>"9107",
    :type=>"user",
    :attributes=>
     {:email=>"lonna@mayert-glover.com",
      :google_id=>"ppn6klzsu1nvfalrw3qk7",
      :google_image_url=>"https://robohash.org/autmodiet.png?size=300x300&set=set1",
      :zip_code=>"47102",
      :summary=>"The mind is not a vessel to be filled but a fire to be kindled.",
      :goal=>"Maintain Weight",
      :availability_morning=>true,
      :availability_afternoon=>true,
      :availability_evening=>false,
      :full_name=>"Ying Nader"}}],
 :meta=>{:gym_member_count=>4}}
```

File: app/controllers/api/v1/gym_memberships/users_controller.rb [hunk 1]
```diff
@@ -0,0 +1,21 @@
+class Api::V1::GymMemberships::UsersController < ApplicationController
+  def index
+    return error_message if params[:yelp_gym_id].blank?
+
+    users = User.users_at_same_gym(params[:yelp_gym_id])
+    options = {}
+    options[:meta] = { gym_member_count: users.size }
+    render json: UserSerializer.new(users, options).serializable_hash,
+           status: :ok
```
Comment (bfl3tch) [inline app/controllers/api/v1/gym_memberships/users_controller.rb:9]: Awesome 👌

File: spec/requests/api/v1/gym_memberships/users/index_request_spec.rb [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+        end
+
+        include_examples 'status code 404'
+      end
+
+      context 'when I provide yelp_gym_id in query params' do
+        let(:gym1_members) { [user1, user2, user3, user5] }
+
+        before { get '/api/v1/gym_memberships/users', params: { yelp_gym_id: gym1 } }
+
```
Comment (bfl3tch) [inline spec/requests/api/v1/gym_memberships/users/index_request_spec.rb:35]: Starting to appreciate this test style more and more

**Review by bfl3tch — APPROVED** [review]

File: app/controllers/api/v1/gym_memberships/users_controller.rb (hunk 1 above)
Comment (tvaroglu) [reply app/controllers/api/v1/gym_memberships/users_controller.rb:9]: ^^^^^^^^

**Review by tvaroglu — APPROVED** [review]
$$$

Nice work!!

**Pull request merged**